architecture/core_objects/builds.adoc

```diff
@@ -204,10 +203,10 @@ example:
 ----
 ====
 
-Once you have the `*Secret*` created, you can add a `PushSecret` field into the
-`Output` section of the `BuildConfig` and set it to the name of the `*Secret*`
-that you created, which in the above example is `*dockerhub*`:
-
+. Add a `PushSecret` field to the `output` section of the `BuildConfig` and
+set it to the name of the `*Secret*` created in step one. As in the above
+example, the following uses `*dockerhub*`:
++
```

Comment:
This is straight-forward, and I can see the name in the above step too, but when is the name established? In the above step too?